source/guide_mumble.rst

```diff
@@ -34,67 +34,61 @@ All relevant legal information can be found here
 
   * https://github.com/mumble-voip/mumble/blob/master/LICENSE
 
-Prerequisites
+Pre-Note
 =============
 
-.. include:: includes/my-print-defaults.rst
+.. note::
+  The official Mumble project decided to no longer provide static murmur server binaries starting with version 1.4.x.
 
-You need a database for mumble:
+  `Some fellow Ubernauts have undertaken efforts <https://github.com/Uberspace/lab/issues/1191>`_ to compile a static murmur binary on their own. You can have a look at it on GitHub_.
 
-.. code-block:: console
+  This guide uses the self-compiled server binary, which currently comes with some limitations, i.e. MySQL is currently not supported as a database. For Ubernauts who have been hosting a Mumble server based on this guide before Mumble v1.4, this might be a breaking change. The default database for a Mumble server is SQLite and requires no further configuration.
+
+  Please read the `release notes <https://github.com/franok/mumble-build-container/releases>`_ of every new release. Releases of the self-compiled server binary may occur more often than the offical Mumble releases. However, no warranty is made regarding up-to-dateness and functional correctness. Consider subscribing to new releases, e.g. by using :lab:`uu-notify <guide_uu-notify>`.
 
-  [isabell@stardust ~]$ mysql -e "CREATE  DATABASE ${USER}_mumble"
-  [isabell@stardust ~]$
 
 Installation
 ============
 
-Check the Mumble_ website or GitHub_ for the feed_ and copy the download link to the ``murmur-static``-file. Replace the URL with the one you just copied.
+Check the `GitHub release section`_ and copy the release tag version of the latest release. Set the variable ``MURMUR_VERSION`` to the version you just copied.
 
-.. note:: the server for mumble is called murmur
+.. note:: The server for mumble is called murmur
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ wget https://github.com/mumble-voip/mumble/releases/download/66.6.6/murmur-static_x86-66.6.6.tar.bz2
-  --2018-10-06 21:14:34--  https://github.com/mumble-voip/mumble/releases/download/66.6.6/murmur-static_x86-66.6.6.tar.bz2
+  [isabell@stardust ~]$ mkdir mumble
+  [isabell@stardust ~]$ cd mumble/
+  [isabell@stardust mumble]$ MURMUR_VERSION=v0.0.0_build-YYYY-000
```

Comment:
This is somehow misleading, because at the source there is only a version, no build with date. Why not simply refer to the latest version?

Reply:
Since this is a self-compiled binary I decided to use a combination of the official-mumble-version + build-number --> `v0.0.0_build-YYYY-000` where `v0.0.0` is the mumble version and `_build-YYYY-000` is the build version of the self-compiled binary.

I updated the git-tag in the github release, it should now no longer be misleading: 
https://github.com/franok/mumble-build-container/releases/tag/v1.4.287_build-2022-001